test/libyul/yulSSAControlFlowGraph/complex.yul

```diff
@@ -53,8 +53,10 @@
 // Entry0 -> Block0_0;
 // Block0_0 [label="\
 // Block 0; (0, max 0)\nLiveIn: \l\
-// LiveOut: \l\nv2 := f(2, 1)\l\
-// pop(v2)\l\
+// LiveOut: \l\nv1 := 2\l\
+// v3 := 1\l\
```

Comment:
Just curious what happened with v2 here, the order of ID assignments is a bit odd, haha

Reply:
Well yeah, it's not strictly monotonous anymore. Not that that is an issue I'd think :)

Reply:
Yeah, it is not an issue. Was just trying to understand the reason for that change haha